src/lib/containers/drug_upload_tool/UserFileGrid.tsx

```diff
@@ -189,58 +198,146 @@ export default class UserFileGrid extends React.Component<
       this.onError('Form Group ID is undefined')
       return <></>
     }
-    if (fileList.length === 0) {
-      return (
-        <h5 className="text-center no-submissions padding-full">
-          You don't have submissions in progress [COPY!]
-        </h5>
+
+    const subhead = <h4>In Progress</h4>
+    const submissionsInProgress = fileList.filter(
+      item => item.submissionStatus.state === StatusEnum.WAITING_FOR_SUBMISSION,
+    )
+    let content = (
+      <h5 className="text-center no-submissions padding-full">
+        You don't have submissions in progress [COPY!]
```

Comment:
Some concern about placeholders making into release (like "[COPY!]" here, or info button that pops up the "What needs to happen here??" alert).  It might be safer to take a best guess for the copy (and leave out the button), and let stakeholders open bugs to request modification.